### PR TITLE
Add documentation for .NET runtime metrics

### PR DESCRIPTION
**Pull request by kevingosse**
### What does this PR do?
Add a documentation page for .NET runtime metrics.

### Preview

https://docs-staging.datadoghq.com/kevin/dotnetruntimemetrics/tracing/runtime_metrics/
https://docs-staging.datadoghq.com/kevin/dotnetruntimemetrics/tracing/runtime_metrics/dotnet/

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/runtime_metrics/dotnet.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      tag: 'Documentation'
+      text: 'Implement Opentracing across your applications.'
+    - link: 'tracing/visualization/'
+      tag: 'Documentation'
+      text: 'Explore your services, resources, and traces'
+---
+
+## Automatic Configuration
+
+Runtime metrics collection can be enabled in .NET Tracer 1.22.0+ with the `DD_RUNTIME_METRICS_ENABLED=true` environment variable.
```
Comment (apigirl) [inline content/en/tracing/runtime_metrics/dotnet.md:22]:
```suggestion
Enable runtime metrics collection in the .NET Tracer 1.22.0+ with the `DD_RUNTIME_METRICS_ENABLED=true` environment variable.
```

File: content/en/tracing/runtime_metrics/dotnet.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Runtime metrics can be viewed in correlation with your .NET services. See the [Service page][1] in Datadog.
```
Comment (apigirl) [inline content/en/tracing/runtime_metrics/dotnet.md:24]:
```suggestion
View runtime metrics in correlation with your .NET services. See the [Service page][1] in Datadog.
```

**Review by apigirl — COMMENTED** [review]
two small grammar feedbacks - making passive voice active

**Review by colin-higgins — APPROVED** [review]

File: content/en/tracing/runtime_metrics/dotnet.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - link: 'tracing/visualization/'
+      tag: 'Documentation'
+      text: 'Explore your services, resources, and traces'
+---
+
+## Automatic Configuration
+
+Enable runtime metrics collection in the .NET Tracer 1.22.0+ with the `DD_RUNTIME_METRICS_ENABLED=true` environment variable.
+
+View runtime metrics in correlation with your .NET services. See the [Service page][1] in Datadog.
```
Comment (zacharycmontoya) [inline content/en/tracing/runtime_metrics/dotnet.md:24]: To maintain consistent wording as other language pages:

```suggestion
Runtime metrics can be viewed in correlation with your .NET services. See the [Service page][1] in Datadog.
```

Comment (zacharycmontoya) [reply content/en/tracing/runtime_metrics/dotnet.md:24]: I just realized that this was recommended by @apigirl . The text `View runtime metrics in correlation with your .NET services. See the [Service page][1] in Datadog.` aligns with the text for the other languages. Which text should we use across languages? I have opened a new PR to add more consistency across the language's runtime metrics pages: #9492

Comment (zacharycmontoya) [reply content/en/tracing/runtime_metrics/dotnet.md:24]: That PR has been merged already, but I can open a new PR for standardizing text across the languages

**Review by lucaspimentel — APPROVED** [review]

File: content/en/tracing/runtime_metrics/dotnet.md [hunk 4]
```diff
@@ -38,6 +38,12 @@ The following metrics are collected by default after enabling .NET metrics.
 
 Along with displaying these metrics in your APM Service Page, Datadog provides a [default .NET Runtime Dashboard][6] with the `service` and `runtime-id` tags that are applied to these metrics.
 
+##  Additional permissions for IIS
+
+On .NET Framework, metrics are collected using performance counters. Users in non-interactive logon sessions (that includes IIS application pool accounts and some service accounts) must be added to the **Performance Monitoring Users** group to access counter data.
+
+IIS application pools use special accounts that do not appear in the list of users. To add them to the Performance Monitoring Users group, look for `IIS APPPOOL\<name of the pool>`. For instance, the user for the DefaultAppPool would be `IIS APPPOOL\DefaultAppPool`.
```
Comment (andrewlock) [inline content/en/tracing/runtime_metrics/dotnet.md:45]: Is it worth being more explicit exactly how to do this, e.g. _how_ do I add a user to a user group? If that's outside the scope of our documentation, could we link somewhere?

**Review by andrewlock — APPROVED** [review]
LGTM, Just a question of whether we should be more explicit about how to do this

**Review by cswatt — APPROVED** [review]
LGTM, can merge after remaining conversations are resolved

**Pull request merged**